src/poggit/errdoc/NotFoundPage.php

```diff
@@ -43,6 +45,21 @@ public function output() {
             <?php Mbd::gaCreate() ?>
             ga('send', 'event', 'Special', 'NotFound', window.location.pathname, {nonInteraction: true});
         </script>
+        <style type="text/css"><?php
+            try {
+                $session = Session::getInstance();
+                if($session === null || !$session->isLoggedIn()) {
+                    readfile(RES_DIR . "style.css");
+                }
+                if($session->getLogin()["opts"]->darkMode ?? false) {
+                    readfile(RES_DIR . "style-dark.css");
+                } else {
+                    readfile(RES_DIR . "style.css");
+                }
+            } catch (\Exception $e){
```

Comment:
What does the catch block here do?

Reply:
Also why not use the usual `inclueCss` function?

Reply:
I used the same for all errdocs to be as safe as possible, think that was already in one of the pages and I just used it for the others